reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md

```diff
@@ -22,167 +21,232 @@ Get-Member [-InputObject <PSObject>] [[-Name] <String[]>] [-MemberType <PSMember
+This example gets the methods and properties of service objects that were extended by using a
+`Types.ps1xml` file or the `Add-Member` cmdlet.
 
-### Example 3: Get extended members of service objects
+```powershell
+Get-Service| Get-Member -View Extended
 ```
-PS> Get-Service| Get-Member -View Extended
-TypeName: System.ServiceProcess.ServiceController
+
+```Output
+    TypeName: System.ServiceProcess.ServiceController
+
 Name MemberType    Definition
 ---- ----------    ----------
 Name AliasProperty Name = ServiceName
 ```
 
-This command gets the methods and properties of service objects that were extended by using the Types.ps1xml file or the Add-Member cmdlet.
-
-The **Get-Member** command uses the *View* parameter to get only the extended members of the service objects.
-In this case, the extended member is the Name property, which is an alias property of the **ServiceName** property.
+The `Get-Member` command uses the **View** parameter to get only the extended members of the service
+objects. In this case, the extended member is the **Name** property, which is an alias property of
+the **ServiceName** property.
 
 ### Example 4: Get script properties of event log objects
-```
-PS> Get-EventLog -Log System | Get-Member -MemberType ScriptProperty
-TypeName: System.Diagnostics.EventLogEntry
-Name    MemberType     Definition
-----    ----------     ----------
-EventID ScriptProperty System.Object EventID {get=$this.get_EventID() -band 0xFFFF;}
+
+This example gets the script properties of event log objects in the System log in Event Viewer.
+
+```powershell
+Get-WinEvent -LogName System -MaxEvents 1 | Get-Member -MemberType NoteProperty
 ```
 
-This command gets the script properties of event log objects in the System log in Event Viewer.
+```Output
+   TypeName: System.Diagnostics.Eventing.Reader.EventLogRecord
 
-The command uses the *MemberType* parameter to get only objects with a value of ScriptProperty for their MemberType property.
+Name    MemberType   Definition
+----    ----------   ----------
+Message NoteProperty string Message=The machine-default permission settings do not grant Local ...
+```
 
-The command returns the EventID property of the **EventLog** object.
+The **MemberType** parameter gets only objects with a value of `NoteProperty` for their
+**MemberType** property.
+
+The command returns the **Message** property of the **EventLogRecord** object.
 
 ### Example 5: Get objects with a specified property
+
+This example gets objects that have a **MachineName** property from a list of cmdlets.
+
+The `$list` variable contains a list of cmdlets to run. The **foreach** statement invokes each
+command and sends the results to `Get-Member`. The **Name** parameter limits the results from
+`Get-Member` to members that have the name **MachineName**.
+
+```powershell
+$list = "Get-Process", "Get-Service", "Get-Culture", "Get-PSDrive", "Get-ExecutionPolicy"
+ForEach ($cmdlet in $A) {& $cmdlet | Get-Member -Name MachineName}
```

Comment:
Shouldn't this be `ForEach ($cmdlet in $list) .....` ?

Reply:
fixed